docs/Reference/API-Methods.md

```diff
@@ -2272,6 +2272,10 @@ You can interact with contracts using [`eth_sendRawTransaction` or `eth_call`].
 `earliest`, or `pending`, as described in
 [Block Parameter](../HowTo/Interact/APIs/Using-JSON-RPC-API.md#block-parameter).
 
+!!! note
+
+    When using `eth_call`, by default, the transaction will not fail if there is an insufficient balance in the sender account. This is done by setting the balance of the account to a very large amount of ether. If you want to enforce balance rules, a new [`strict` parameter](API-Objects.md#transaction-call-object) in the transaction call object can be set to `true`.
```

Comment:
According to our guidelines we should try (where possible) to avoid using "will", also avoid using "a new", since in a few months time it will no longer be new, so maybe:

"By default, `eth_call` does not fail if the sender account has an insufficient balance. This is done by setting the balance of the account to a large amount of ether. To enforce balance rules, set the [`strict` parameter](API-Objects.md#transaction-call-object) in the transaction call object to `true`.